public/js/dashboardCategories.js

```diff
@@ -4,47 +4,39 @@
  * @return {JSON} All category entry information.
  */
 async function getAllCategories() {
-  let responseData = await fetch('/data/categories');
-  let responseDataJSON = await responseData.json();
-  let allCategories = responseDataJSON.map(
-    function (category) {
-      const obj = {
-        ID: category,
-        name: category.replace(/-/g, " "),
-      }
-      return obj;
-    });
+  const responseData = await fetch('/data/categories');
+  const responseDataJSON = await responseData.json();
+  const allCategories = responseDataJSON.map(function (category) {
+    const obj = {
+      ID: category,
+      name: category.replace(/-/g, ' '),
+    };
+    return obj;
+  });
   return allCategories;
 }
 
 /**
- * @param {string} organizationID ID of organization requesting categories.
+ * Gets all categories that have been accepted by the currently logged in organization.
  * @return {JSON} Object containing all categories used by an organization.
  */
-function getAllAcceptedCategories(organizationID) {
-  // TODO: Get an organizations accepted cagtegories from database
-  const allAcceptedCategories = [
-    {
-      name: 'Clothes',
-      ID: 'org-name-clothes',
-      instructions: [
-        'review quality checklist',
-        'unlock donation locker (code: 5248)',
-        'place in donation locker',
-      ],
-      quality: ['no holes', 'gently used', 'washed recently'],
-    },
-    {
-      name: 'Food',
-      ID: 'org-name-food',
-      instructions: [
-        'review quality checklist',
-        'place in plastic/paper bag',
-        'bring inside during open hours',
-      ],
-      quality: ['expiration date > 6 months from current date', 'no holes', 'labels on cans'],
-    },
-  ];
+async function getAllAcceptedCategories() {
+  const memberData = await (await fetch('/data/member')).json();
+  const organizationData = await (await fetch(`/data/organization/member/${memberData.id}`)).json();
+  const responseData = await (
+    await fetch(`/data/acceptedcategories/organization/${organizationData.id}`)
+  ).json();
```

Comment:
Not positive, but I think you may only need `(await fetch(url)).json()` for these lines instead of having the outer await.

Reply:
unfortunately it's required, do you think making variables for the two awaits would be cleaner?